DirectProgramming/DPC++FPGA/Tutorials/Features/scheduler_target_fmax/third-party-programs.txt

```diff
@@ -0,0 +1,253 @@
+oneAPI Code Samples - Third Party Programs File
```

Comment:
You don't need this file in your tutorial, please delete it.

Reply:
Will do.